### PR TITLE
Add up-for-grabs .yml file for Light Table. Addresses #2366.

### PR DESCRIPTION
**Pull request by chenhowa**
This is a basic .yml file for up-for-grabs.net. If it is okay, I can submit a PR for it to the up-for-grabs repo as well. Thanks!

### CODE REVIEW

**Comment by chenhowa** [comment]
#2366

**Comment by Mouvedia** [comment]
Hi @chenhowa,

I would have added _editor_ as a tag as well; apart from LGTM.
You can PR under `_data/projects/` on https://github.com/up-for-grabs/up-for-grabs.net and close this one.

**Comment by chenhowa** [comment]
Sounds good.

File: light-table.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name: Light Table
+desc: Light Table is a next generation code editor that connects you to your creation with instant feedback. Light Table is very customizable and can display anything a Chromium browser can.
+site: http://lighttable.com/
+tags:
+- ide
+- clojure
+- clojurescript
+- electron
+upforgrabs:
+    name: good first issue
```
Comment (Mouvedia) [inline light-table.yml]: It should be prepended by 2 spaces not 4. I don't know if it will make a difference though.

Comment (chenhowa) [reply light-table.yml]: Okay, I'll fix that in my PR to up-for-grabs.

**Pull request closed without merging**